src/tests/double_array_structure.rs

```diff
@@ -98,3 +98,76 @@ fn test_input_order() {
     assert_eq!(pma_sorted.states, pma_unsorted.states);
     assert_eq!(pma_sorted.outputs, pma_unsorted.outputs);
 }
+
+#[test]
+fn test_n_blocks_1_1() {
+    let mut patterns = vec![];
+    // state 0: reserved for the root state
+    // state 1: reserved for the dead state
+    // base = 0xff; fills 0x02..=0x7f
+    for i in 0x00..=0xfd {
+        let pattern = vec![i];
+        patterns.push(pattern);
+    }
+    let pma = DoubleArrayAhoCorasick::new(patterns).unwrap();
+    assert_eq!(255, pma.num_states());
+    assert_eq!(256, pma.states.len());
+}
```

Comment:
```suggestion
    assert_eq!(pma.states[0].base().unwrap().get(), 0xfe);
}
```
It would better to add to check `base` value.